Library/ViewModels/BackingViewModel.swift

```diff
@@ -238,45 +250,118 @@ public final class BackingViewModel: BackingViewModelType, BackingViewModelInput
-    case .collected:
-      return Strings.project_view_pledge_status_collected()
-    case .dropped:
-      return Strings.project_view_pledge_status_dropped()
-    case .errored:
-      return Strings.project_view_pledge_status_errored()
-    case .pledged:
-      return Strings.project_view_pledge_status_pledged()
-    case .preauth:
-      fatalError()
+private func statusDescString(for backing: Backing, project: Project, backerIsCurrentUser: Bool)
+  -> NSAttributedString {
+
+  var string = ""
+  switch backing.status {
+  case .canceled:
+    string = backerIsCurrentUser
+      ? Strings.Your_pledge_was_canceled_or_the_creator_canceled()
+      : Strings.Either_the_pledge_or_the_project_was_canceled()
+  case .collected:
+    string = backerIsCurrentUser
+      ? Strings.Your_payment_method_was_successfully_charged()
+      : Strings.Payment_method_was_successfully_charged()
+  case .dropped:
+    string = backerIsCurrentUser
+      ? Strings.Your_pledge_was_dropped()
+      : Strings.Pledge_was_dropped()
+  case .errored:
+    string = Strings.There_was_a_problem_with_this_payment()
+  case .pledged:
+    let isCanceledOrFailed = (project.state == .canceled || project.state == .failed)
+    string = backerIsCurrentUser
+      ? (isCanceledOrFailed
+          ? Strings.Your_pledge_was_canceled_or_the_creator_canceled()
+          : Strings.Youve_pledged_to_support_this_project()
+        )
+      : (isCanceledOrFailed
+          ? Strings.Either_the_pledge_or_the_project_was_canceled()
+          : Strings.Backer_has_pledged_to_this_project()
+        )
+  case .preauth:
+    fatalError()
+  }
+
+  if backing.status == .collected {
+    return NSAttributedString(string: string, attributes: [
+      NSFontAttributeName: UIFont.ksr_headline(size: 13),
+      NSForegroundColorAttributeName: UIColor.ksr_text_green_700
+    ])
+  } else {
+    return NSAttributedString(string: string, attributes: [
+      NSFontAttributeName: UIFont.ksr_subhead(size: 13),
+      NSForegroundColorAttributeName: UIColor.ksr_text_navy_500
+    ])
   }
 }
+
+private func pledgeTitle(for backing: Backing, project: Project, backerIsCurrentUser: Bool)
+  -> NSAttributedString {
+
+  let date = Format.date(secondsInUTC: backing.pledgedAt, dateStyle: .long, timeStyle: .none)
+
+  let titleString = backerIsCurrentUser
+    ? Strings.You_pledged_on_date(pledge_date: date)
+    : Strings.Pledged_on_date(pledge_date: date)
+
+  return titleString.simpleHtmlAttributedString(
+    base: [
+      NSFontAttributeName: UIFont.ksr_subhead(size: 13),
+      NSForegroundColorAttributeName: UIColor.black
+    ],
+    bold: [
+      NSFontAttributeName: UIFont.ksr_headline(size: 15),
+      NSForegroundColorAttributeName: UIColor.black
+    ])
+    ?? NSAttributedString(string: "")
+}
+
+private func rewardTitle(for reward: Reward?, project: Project, backerIsCurrentUser: Bool)
+  -> NSAttributedString {
+
+  guard let reward = reward else { return NSAttributedString(string: "") }
```

Comment:
We only seem to be using `reward` to grab the delivery estimate. Can we eliminate the variable?

``` swift
guard let estimatedDate = reward?.estimatedDeliveryOn else { return .init() }
```